lib/coder.js

```diff
@@ -1,109 +1,114 @@
-var vlq = require('./vlq');
-var fields = ['generatedColumn', 'source', 'originalLine', 'originalColumn', 'name'];
+'use strict';
 
-module.exports = Coder;
-function Coder() {}
-
-Coder.prototype.decode = function(mapping) {
-  var value = this.rawDecode(mapping);
-  var output = {};
-
-  for (var i=0; i<fields.length;i++) {
-    var field = fields[i];
-    var prevField = 'prev_' + field;
-    if (value.hasOwnProperty(field)) {
-      output[field] = value[field];
-      if (typeof this[prevField] !== 'undefined') {
-        output[field] += this[prevField];
+const vlq = require('./vlq');
+
+const FIELDS = ['generatedColumn', 'source', 'originalLine', 'originalColumn', 'name'];
+
+class Coder {
+  constructor() {}
```

Comment:
this is the default, no need for it to be specified if its an empty function

Reply:
fixed